Resources/Prototypes/_NF/Entities/Structures/Storage/construction_box.yml

```
@@ -0,0 +1,28 @@
+- type: entity
+  id: ConstructionBox
+  name: construction box
+  description: A large storage container for holding materials.
+  parent: PlantBox
+  components:
+  - type: Damageable
+    damageContainer: Inorganic
+    damageModifierSet: Metallic
```

```suggestion
    damageModifierSet: Metallic
  - type: Destructible
    thresholds:
    - trigger:
        !type:DamageTrigger
        damage: 450
      behaviors:
      - !type:DoActsBehavior
        acts: [ "Destruction" ]
    - trigger:
        !type:DamageTrigger
        damage: 300
      behaviors:
      - !type:EmptyAllContainersBehaviour
      - !type:SpawnEntitiesBehavior
        spawn:
          SheetSteel1:
            min: 3
            max: 5
      - !type:DoActsBehavior
        acts: [ "Destruction" ]
```
Minor thing - box should probably spawn steel on destruction since it's made of metal.